### DISCUSSION TITLE
TuringCourses/math_phys/physics2/

### DISCUSSION THREAD

**Discussion by giscus** [ZJU-Turing/TuringCourses#50] [General]
# TuringCourses/math_phys/physics2/

图灵班学习指南

https://zju-turing.github.io/TuringCourses/math_phys/physics2/

**Comment by 626lt** [comment]
22级方明虎老师班上课体验：
由于到课人数长期低迷，虎哥的点名频率很高，一个学期达到了11次之多，能到课还是尽量到课。整体课程大概分为期中考前电磁学，期中后光学量子力学。22级量子力学老师仅用一次课介绍完，但在期末考试中出现了不少的默写题（薛定谔方程等）。而且期末考试几乎没有怎么考察期中之前的内容，很多都是概念性的东西，难度相对期中考而言是简单一些。对于复习的建议是期中重点拟合历年卷，期末重视后半程的内容。尽管考试和平时上课的内容难度都较大，但是相对于隔壁王业伍老师的“阴着坏”（x）而言，虎哥的卷子还是相对合理而且考察重点和上课内容也是比较吻合的。在给分方面，整体给分应该是挺捞的，结束后群里的谢谢老师相当多。还可以参加大学生物理竞赛获得加分，在22级外教班似乎没有这个加分。不管你的物理基础好不好，我都非常建议选择虎哥的课，真的讲的很细致会很有收获。
